### ISSUE TITLE
Newer model switch does not seem supported

### ISSUE THREAD

**Issue by jeanluc-one** [angonz/tech2xl#4]
Hello Andres,

We have several Cisco CAT9K series switches (C9200L, C9300) and the code always gives the same error when we try to use the show tech or show interfaces status, C2960 series and C3560 work fine.

python tech2xl.py export.xls show-tech.txt
tech2xl v1.5
Traceback (most recent call last):
  File "tech2xl.py", line 422, in <module>
    if intinfo[name][item]['Description'] == '':
KeyError: None

[show tech.txt](https://github.com/angonz/tech2xl/files/5383642/show.tech.txt)
[show int status.txt](https://github.com/angonz/tech2xl/files/5383643/show.int.status.txt)

**Comment by mitchbradford** [comment]
@jeanluc-one  did you get this working? Works fine on my 9200L-48P-4X

**Comment by jeanluc-one** [comment]
No, just tested again. It does not work with C9300-24UX and C9300-48UN. It still gives me an error on line 422.

**Comment by jeanluc-one** [comment]
Hello @mitchbradford do you have an update on this issue? If needed we can arrange a session to check the script with the model switch.

**Comment by jeanluc-one** [comment]
Hello all, is there any update on this? We would love to get this working.

**Comment by angonz** [comment]
Hi @jeanluc-one!  Sorry... I haven't used this script since long ago... I have made an update, please check and let me know if it works now.

**Comment by jeanluc-one** [comment]
Hello @angonz 

unfortunately it’s still not working. I believe this has to do with the introduction of multi gigabit, twenty five, forty and hunderd gigabit interfaces.

Do you want me to supply a tech support for one of the switches?

**Comment by lkwashok** [comment]
Hello Angonz, While using script through index it open in browser, and if I give file input and upload it says file not found error,  I am using windows as well Python 3 also I and suggested packeg too need ur help
